src/components/InboxScreen.stories.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import { Provider } from 'react-redux';
 
-import { PureInboxScreen } from './InboxScreen';
+import InboxScreen, { PureInboxScreen } from './InboxScreen';
 import { defaultTasks } from './TaskList.stories';
 
+export default {
+  Component: InboxScreen,
```

Comment:
lower case

Reply:
Interesting, although the prop types of the pure component may not be quite right